src/flow.jl

```diff
@@ -121,24 +121,31 @@ function initialize_surfaceflow_river(
 
     n_river = ncread(
         nc,
-        config.input,
-        "lateral.river.n";
+        config.input.lateral.river,
```

Comment:
By removing the `lateral.river` part, this part now does not end up in the logging. In `ncread` this String is the `parameter` argument, which is also used for info log messages in `ncread`. Now, it is for example difficult to differentiate between `n` for `land` and `river` domain. If I understand correctly, using `lateral.river.bankfull_depth` (below) is not possible in combination with an alias. But, in case an alias is not used I prefer to use the full info here. Or perhaps you've another idea how we can deal with this @visr ?

Reply:
Can you check ad058f807141ed12f8c4876186a26b729d378fc6? Should be good to go now. I don't think adding a test is needed, though I did confirm that the old name works and gives an error now.

Reply:
Looks good, I also did change a coupe of soil parameters to full parameter name where an alias is used, see https://github.com/Deltares/Wflow.jl/pull/199/commits/06edf23e3bbbe17abc649970b0d07ecf7c738c88.